src/MzXML.jl

```diff
@@ -241,7 +241,7 @@ function Base.copyto!(dest::AxisArray, src::Vector{<:MSscan})
     axmz, axt = AxisArrays.axes(dest, Axis{:mz}), AxisArrays.axes(dest, Axis{:time})
     fill!(dest, 0)
     for scan in src
-        j, ft = val2index(axt, scan.retentionTime)
+        j, ft = val2index(axt, scan.retentionTime.val)
```

Comment:
axt is a range like 0:1:10, scan.retentionTime is a "Quatity" like 2s (where s is second). then, operations like "scan.retentionTime - 2" would throw bugs. scan.retentionTime.val will return the value of this "Quantity".